mmseg/models/backbones/vit.py

```diff
@@ -261,6 +263,7 @@ def __init__(self,
                  act_cfg=dict(type='GELU'),
                  norm_eval=False,
                  final_norm=False,
+                 final_reshape=True,
```

Comment:
```suggestion
                 out_shape=`NCHW`,
```
We may rename it as `out_shape`.